.circleci/config.yml

```diff
@@ -1,79 +1,88 @@
-          javac -cp target/liboqs-java.jar examples/SigExample.java &&
-          javac -cp target/liboqs-java.jar examples/RandExample.java )
-    - run:
-        name: Run KEMs, Signatures and Rand examples
-        command: >
-          .circleci/git_no_checkin_in_last_day.sh || (
-          export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/usr/local/lib" &&
-          java -Djava.library.path=target/ -cp target/liboqs-java.jar:examples/ KEMExample &&
-          java -Djava.library.path=target/ -cp target/liboqs-java.jar:examples/ SigExample &&
-          java -Djava.library.path=target/ -cp target/liboqs-java.jar:examples/ RandExample )
-    - store_test_results:
-        path: target/surefire-reports
-    - store_artifacts:
-        path: target/liboqs-java.jar
+version: 2.1
 
 jobs:
-  circleci-openjdk-9:
-    <<: *myjob
-    environment:
-      IMAGE: circleci/openjdk:9-jdk
-      SUDO: sudo
-  debian-buster-amd64:
-    <<: *myjob
-    environment:
-      IMAGE: openquantumsafe/ci-debian-buster-amd64:latest
-      SUDO: ''
-  ubuntu-bionic-x86_64:
-    <<: *myjob
-    environment:
-      IMAGE: openquantumsafe/ci-ubuntu-bionic-x86_64:latest
-      SUDO: ''
+  ubuntu_bionic:
+    description: A template for running liboqs-java tests on the OQS x64 Ubuntu Bionic Docker VM
+    docker:
+      - image: openquantumsafe/ci-ubuntu-bionic-x86_64:latest
+        auth:
+          username: $DOCKER_LOGIN
+          password: $DOCKER_PASSWORD
+    steps:
+      - checkout # change this from "checkout" to "*localCheckout" when running CircleCI locally
+      - run:
+          name: Clone liboqs
+          command: git clone --branch main --single-branch --depth 1 https://github.com/open-quantum-safe/liboqs.git
+      - run:
+          name: Build liboqs
+          command: mkdir build && cd build && cmake .. -G"Ninja" -DBUILD_SHARED_LIBS=ON -DOQS_BUILD_ONLY_LIB=ON && ninja install
+          working_directory: liboqs
+      - run:
+          name: Resolve all maven project dependencies
+          command: mvn dependency:go-offline
+      - run:
+          name: Build liboqs-java and run tests
+          command: export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/usr/local/lib" && mvn package
+      - run:
+          name: Compile KEM, Signatures and Rand examples
+          command: >
+            javac -cp target/liboqs-java.jar examples/KEMExample.java &&
+            javac -cp target/liboqs-java.jar examples/SigExample.java &&
+            javac -cp target/liboqs-java.jar examples/RandExample.java
+      - run:
+          name: Run KEM, Signatures and Rand examples
+          command: >
+            export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/usr/local/lib" &&
+            java -Djava.library.path=target/ -cp target/liboqs-java.jar:examples/ KEMExample &&
+            java -Djava.library.path=target/ -cp target/liboqs-java.jar:examples/ SigExample &&
+            java -Djava.library.path=target/ -cp target/liboqs-java.jar:examples/ RandExample
+
+  openjdk-9:
```

Comment:
Should we change the name to `openjdk-11` from `openjdk-9` since we use `cimg/openjdk:11.0.9`? (also in line 86)